src/data/sym/sym2.lean

```diff
@@ -244,16 +240,14 @@ A predicate for testing whether an element of `sym2 α` is on the diagonal.
 def is_diag : sym2 α → Prop :=
 lift ⟨eq, λ _ _, propext eq_comm⟩
 
-lemma is_diag_iff_eq {x y : α} : is_diag ⟦(x, y)⟧ ↔ x = y :=
-iff.rfl
+lemma mk_is_diag_iff {x y : α} : is_diag ⟦(x, y)⟧ ↔ x = y := iff.rfl
```

Comment:
```suggestion
lemma is_diag_mk_iff {x y : α} : is_diag ⟦(x, y)⟧ ↔ x = y := iff.rfl
```

Reply:
Oh, I guess this doesn't match diag_is_diag...

Reply:
Yeah... I just picked one way and stuck with it.